fitsio/fitslib.py

```diff
@@ -1760,7 +1760,9 @@ def _verify_column_data(self, colnum, data):
         """
         this_dt = data.dtype.descr[0]
 
-        if len(data.shape) > 1:
+        if len(data.shape) > 2:
+            this_shape = data.shape[1:]
+        elif len(data.shape) == 2 and data.shape[1] > 1:
```

Comment:
This is the main one @esheldon

Reply:
I understand this now.  I thought we were dealing with a different issue.

Reply:
this makes sense I think